### DISCUSSION TITLE
Allow for persistent person identifiers such as ORCID iD to be added to GitHub profiles

### DISCUSSION THREAD

**Discussion by errortryagain** [community/community#12543] [Other Feature Feedback, Questions, & Ideas]
I can think of at least the following ways, in which unique persistent person identifiers such as ORCID iD, Wikidata ID or others could be an asset to GitHub profiles.

1) Let GitHub users voluntarily add any of these IDs in a standardized way (i.e. either as mere ID in combination with the scheme, e.g. "0000-1234-5678-0000" with scheme "ORCID iD" so that it can then be mapped to "https://orcid.org/0000-1234-5678-0000", or else, directly as such a link) to their profiles. That way, a person's GitHub output can be correctly connected to them and their other research output (data, publications, presentations etc.) for example through linked open data projects.

2) Allow for a login via an ORCID iD (additional login option next to the GitHub credentials option) so that the output created on GitHub may also be exported to a person's ORCID record. 

Does anybody else see a necessity for that? Data/PID managers, data librarians, Wikidata editors, perhaps?

**Comment by musoke** [comment]
The first request is available now. See https://info.orcid.org/orcid-and-github-sign-memorandum-of-understanding/ and https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#adding-your-orcid-id-to-your-profile.

**Comment by JibachhhMahto764** [comment]
Enabling ORCID iDs on GitHub profiles allows seamless linking of code contributions to researchers, enhancing transparency and recognition in academia. This feature streamlines attribution, benefiting both individual contributors and the broader academic community by ensuring accurate credit for open-source work on the platform.
